### DISCUSSION TITLE
Ollama 3.0

### DISCUSSION THREAD

**Discussion by peuportier** [open-webui/open-webui#4124] [General]
Ollama released version 3.0 - [Ollama now supports tool calling with popular models](https://ollama.com/blog/tool-support) - This can also will made openwebui more smooth to using tools ?

**Comment by justinh-rahb** [comment]
In the short term, no it doesn't change anything about how our current Tools are implemented. Longer-term we may be able to use this to improve accuracy and performance, but fundamentally it's just an alternative method of doing what we're already doing.

**Comment by peuportier** [comment]
@justinh-rahb  Thanks you for your time and explanation. Appreciated ! Wish you a good day
